### ISSUE TITLE
Raidboss - Short stack during Hello World

### ISSUE THREAD

**Issue by HailZaros** [quisquous/cactbot#313]
Hey,
This is completely unrelated to the 4.5 patch update, rather something I've just been too lazy to ask.
I've been trying to make it so there's a "Short stack on <playerName>" only for the first DPS player who got the 8 second stack debuff, so it's easier to spot and call out/heal.
Currently I have it modified to show whoever has the short stack no matter what, as well as the personal role, but was wondering if there is perhaps a more elegant way to make it show purely for the first DPS?
Both names and Job names are fine for this.
Thanks!

**Comment by quisquous** [comment]
I think the way to do this would be to make a second trigger for short stack that gets sent to everybody and store something in `data`, like `data.shortStack = name;`.  You could then have a conditional on the trigger such that it only triggers `if !data.shortStack`.

I can also look into adding something like this.  This is a good idea!

**Comment by quisquous** [comment]
Yeah, that condition would work.  Another option would be to add a `suppressSeconds: 40` as well so that it only happens once per hello world.

**Comment by HailZaros** [comment]
Wouldn't that block the message for the long stack marker, though?
As in, if you have long stack - you won't get the message for you having it, if the short stack were to apply first, since they're the same debuffs (and therefore the same regex), just that each has a different duration?

I'd assume it won't show up, and even if it does - it could kind of be inconsistent depending on which log line shows up first, right?

Either way, I would only be able to test if the condition I've made is working during next weeks clears, so if you want to whip up a cleaner version of it, feel free to.
Otherwise, i'll likely just submit a pull request with it after I've confirmed it works.

Thanks again for the help! 👍
